src/templates/events/EventsYouMayLike.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import EventListingCard from './EventListingCard'
+import { Consumer } from '../../components/AppContext'
+import { Container, Row, Column } from '../../components/grid/grid'
+
+const ViewAll = styled.a`
+  color: ${props => props.theme.colors.indigo};
+  display: block;
+  font-family: ${props => props.theme.fonts.title};
+  font-size: 0.8em;
+  letter-spacing: 0.1px;
+  padding-top: 5px;
+  text-align: right;
+  text-decoration: none;
+  width: 100%;
+`
+
+const FlexColumn = styled(Column)`
+  display: flex;
+`
+
+const filterEventsYouMayLike = (events, eventId) => {
+  const filteredEvents = events.filter(event => {
+    if (event.node.id === eventId) return false
+
+    const timeNow = moment().format('LLLL')
+    const eventStartTime = moment(event.node.startTime).format('LLLL')
+    return moment(eventStartTime).diff(timeNow, 'days') > 0
```

Comment:
A few things here:

 - This line is logging a warning because moment doesn't like diffing with formatted strings. So `timeNow` can be replaced with `moment`.
- Similarly, `eventStartTime` can be replaced with `event.node.startTime`
- And then you can delete the two lines above
- Seems like diffing by `'days' `returns a floored integer, so this will never allow users to see events that are occurring less than 24 hours away. That's probably a bad idea. Maybe we should change this from `'days'` to `'hours'` or even `'minutes'`.